doc/tutorials/partition-mode.rst

```diff
@@ -0,0 +1,332 @@
+                        .type     =  E820_TYPE_RESERVED
+                    },
+
+                    {   /* PCI hole to 4G */
+                        .baseaddr =  0xe0000000U,
+                        .length   =  0x20000000U,
+                        .type     =  E820_TYPE_RESERVED
+                    },
+                }
+
+
+   E.   Do modifications to ``mptable_vm(n)`` (where n stands for each UOS VM). Each UOS
+        VM has a mptable defined in ``arch/x86/guest/mptable.c``.
+
+         Make sure ``VM1_NUM_CPUS`` and ``VM2_NUM_CPUS`` are defined to the same
+         values as they are in ``partition\vm_description.c``.
+
+         ``proc_entry_array`` member in struct ``mptable_info`` is a flexible array.
+         Each cpu of the UOS should be represented by a ``proc_entry`` in ``proc_entry_array``.
+
+        ``apic_id`` in the ``proc_entry`` should be same as physical APIC ID
+        of physical CPU that belongs to the UOS VM.
+
+        For the following configuration defined in ``partition/vm_description.c``,
+
+               .. code-block:: none
+
+                  #define VM1_NUM_CPUS    4U
+
+                  /* Logical CPU IDs assigned to this VM */
+                  uint16_t VM1_CPUS[VM1_NUM_CPUS] = {0U, 2U, 4U, 6U};
+
+         ``proc_entry_array`` in ``mptable_vm1`` should have 4 ``proc_entry``
+         members and the ``apic_id`` of each ``proc_entry`` should be corresponding
+         to the physical APIC id of the physical processors ``{0U, 2U, 4U, 6U}``.
+
+Building ACRN with partition mode enabled
+*****************************************
+
+For MRB board, change directory to ``acrn-ebtool`` (the top-level directory where
+the ACRN hypervisor and ACRN kernel code reside), then:
+
+#. Build the UOS kernel and rootfs.
+
+   .. code-block:: none
+
+      $ make uos_kernel uos_rootfs
+
+   The build results are found under the ``out/uos_kernel`` and ``out/uos_rootfs`` directories.
+
+
+
+#. Build and flash the ACRN hypervisor.
+
+   .. code-block:: none
+
+      $ make hypervisor stitch sos_boot
+      $ make flash_acrn
+
+
+Copying UOS rootfs to storage device
+************************************
+
+Connect the storage device to your computer and identify it using the ``lsblk`` command.
+Throughout this procedure, the target storage device name used is ``/dev/sdX``, substitute 
+sdX with the correct device for your storage device.
+
+.. code-block:: none
+
+     $ sudo mount out/uos_rootfs.img /mnt/uos_rootfs/
+     $ sudo umount /dev/sdX
+     $ sudo mkfs.ext4 /dev/sdX
+     $ sudo mkdir /mnt/sdX
+     $ sudo mount /dev/sdX /mnt/sdX
+     $ cd /mnt/sdX
+     $ sudo rm -fr *
+     $ sudo cp -ax /mnt/uos_rootfs/* .
+     $ cd ~
+     $ sync
+     $ sudo umount /mnt/sdX
+
+
+Accessing UOS console from ACRN console
+***************************************
+
+Once ACRN is booted by the bootloader, ACRN starts the UOS VMs. Each UOS VM console can be accessed by the command ``sos_console x``.
```

Comment:
All VMs are referred to as UOS but the command to enter the console is `sos_console`, is this expected?